lib/core/src/display/world.rs

```diff
@@ -226,7 +193,7 @@ impl World {
     pub fn new(world_data: WorldData) -> Self {
         let rc = Rc::new(RefCell::new(world_data));
         let out = Self {rc};
-        init_global_variables(&out);
+//        init_global_variables(&out);
```

Comment:
So, if we resign from global variables, I would remove this line instead of commenting it